src/main/kotlin/eu/europa/ec/eudi/pidissuer/PidIssuerApplication.kt

```
@@ -545,12 +547,23 @@ fun beans(clock: Clock) = beans {
     bean {
         CreateCredentialsOffer(ref(), credentialsOfferUri)
     }
+    bean {
+        val knownVct = runCatching {
```

@atsant Some comments:

1. You don't have a single known vct. You have multiple.
2. It's not vctEudi, it the PID vct
3. It's not classPathEudi, it's the PID Type Metadata
4. In case the classpath resource doesn't exist, let it fail, do not let the application start when you have detected configuration errors